docker-hub/publish/insights-analytics.md

```diff
@@ -161,20 +159,20 @@ The following table describes the rules applied for determining intent behind
-| GET            | digest    | N/A                                                             | Pull by digest   | User already has all layers existing on local machine and/or the image is single-arch                          |
-| HEAD           | digest    | N/A                                                             | Pull by digest   | User already has all layers existing on their local machine.                                                   |
-| GET            | digest    | GET by different digest                                         | Pull by digest   | Image is multi-arch                                                                                            | The second GET by digest must be different from the first                                                                                                                                                                                                                                      |
-| HEAD           | digest    | GET by same digest                                              | Pull by digest   | Image is single arch and/or image is multi-arch but some part of the image already exists on the local machine |
-| HEAD           | digest    | GET by same digest, then a second GET by different digest       | Pull by Digest   | Image is multi-arch                                                                                            |
+| HEAD           | tag       | N/A                                                             | Version check    | User already has all layers existing on local machine                                                          | This is similar to the use case of a pull by tag when the user already has all the image layers existing locally, however, it differentiates the user intent and classifies accordingly.                                                                                              |
+| GET            | tag       | N/A                                                             | Pull by tag      | User already has all layers existing on local machine and/or the image is single-architecture                          |
+| GET            | tag       | Get by different digest                                         | Pull by tag      | Image is multi-architecture                                                                                            | Second GET by digest must be different from the first.                                                                                                                                                                                                                                         |
```

Comment:
the term 'multi-arch' is very commonly used, I'm not sure we should switch to the abbreviated form (and therefore also keep "single-arch" for consistency)

Reply:
Awwright then, gosh darn that jargon ;)